### PR TITLE
Fix/UI 2

### PR DESCRIPTION
**Pull request by snadrus**
https://github.com/filecoin-project/curio/issues/84 

Fix:
 - remove 24h tasks column from machine(node) table and add 2 new columns
      Layers used by node
     Tasks supported by the node
- We should rename "config layer available" to "Source layer" in Actor summary.

### CODE REVIEW

**Comment by LexLuthr** [comment]
@snadrus Can we have some screenshots for this? It is hard to understand what rendering will look like?

File: cmd/curio/rpc/rpc.go [hunk 1]
```diff
@@ -244,7 +245,7 @@ func (p *CurioAPI) LogSetLevel(ctx context.Context, subsystem, level string) err
 	return logging.SetLogLevel(subsystem, level)
 }
 
-func ListenAndServe(ctx context.Context, dependencies *deps.Deps, shutdownChan chan struct{}) error {
+func ListenAndServe(ctx context.Context, dependencies *deps.Deps, activeTasks []harmonytask.TaskInterface, shutdownChan chan struct{}) error {
```
Comment (LexLuthr) [inline cmd/curio/rpc/rpc.go:248]: Can we avoid doing all this if we just add these details to the machine details table?

Comment (snadrus) [reply cmd/curio/rpc/rpc.go:248]: No, this is about knowing which tasks we can get the SP_ID details for and which are "whole system" tasks. This is not a database matter but an introspection of which enabled tasks support this interface.

Comment (LexLuthr) [reply cmd/curio/rpc/rpc.go:248]: Makes sense.

**Review by LexLuthr — APPROVED** [review]

**Pull request merged**